rdesktop.c

```diff
@@ -669,6 +671,16 @@ main(int argc, char *argv[])
 					p++;
 				}
 
+				if (*p == '@')
+				{
+					g_dpi = strtol(p + 1, &p, 10);
+					if (g_dpi <= 0)
+					{
+						error("invalid geometry\n");
```

Comment:
Could you change this to logger(Core, Error, whatever you want to show to user)? 
e.g.:
logger(Core, Error, "Wrong DPI value (%d)", g_dpi);

Reply:
Well I could but that check is intended to catch @letters so %d is silly.

Reply:
Ok.
The point is to use the logger() instead of error().
What you're going to output to user is up to you here actually